docker/Dockerfile.http

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04
```

Comment:
Thought about using Alpine? Ubuntu is a pretty big OS.

Reply:
I would rather use standard maven image from docker hub and replace lines 8-14 with:
`FROM maven:3.8-jdk-11-slim`
It brings a discussion about Alpine vs Debian Slim. I prefer Debian. Alpine doesn't use standard `glibc` which can cause some compatibility issues. We can probably create a new story to fix it.

Reply:
Definitely using an application-specific image as a basis is better than reaching upwards to an operating system—and that means `maven`.

(In my own Docker experience I've used Alpine with its `musl` libc versus `glibc` for several years now and have never encountered an issue.)